include/cuda/std/detail/libcxx/include/span

```diff
@@ -118,32 +121,29 @@ template<class T, size_t N>
 template<class T, size_t N>
     span(const array<T, N>&) -> span<const T, N>;
 
-template<class Container>
-    span(Container&) -> span<typename Container::value_type>;
-
-template<class Container>
-    span(const Container&) -> span<const typename Container::value_type>;
+template<class R>
+    span(R&&) -> span<remove_reference_t<ranges::range_reference_t<R>>>;
 
 } // namespace std
 
 */
 
+#ifndef __cuda_std__
 #include <__config>
 #include <cstddef>      // for ptrdiff_t
 #include <iterator>     // for iterators
 #include <array>        // for array
 #include <type_traits>  // for remove_cv, etc
 #include <cstddef>      // for byte
+#endif // __cuda_std__
 
 #if defined(_LIBCUDACXX_USE_PRAGMA_GCC_SYSTEM_HEADER)
 #pragma GCC system_header
 #endif
 
 _LIBCUDACXX_BEGIN_NAMESPACE_STD
 
-#if _LIBCUDACXX_STD_VER > 17
```

Comment:
If I remember correctly, mdspan is C++14 only, so maybe we want to restrict this also to C++14?

Reply:
Yeah, I don't think we need to go through any efforts to backport this (or anything else) to C++11.